tests/cypress/e2e/ITILObject/ticket_form.cy.js

```diff
@@ -107,4 +107,50 @@ describe("Ticket Form", () => {
             });
         });
     });
+
+    it('Validation Template', { retries: 0 }, () => {
+        const rand = Math.floor(Math.random() * 1000);
+        cy.createWithAPI('Ticket', {
+            name: 'apple',
+            content: 'apple',
+        }).as('ticket_id');
+        cy.createWithAPI('ITILValidationTemplate', {
+            name: `test ${rand}`,
+            content: 'test content',
+            entities_id: 1,
+        }).as('validationtemplates_id');
+        cy.createWithAPI('ITILValidationTemplate', {
+            name: `test no approver ${rand}`,
+            content: 'no approver test content ',
+            entities_id: 1,
+        }).as('validationtemplates_id2');
+
+        cy.get('@ticket_id').then((ticket_id) => {
+            cy.get('@validationtemplates_id').then((validationtemplates_id) => {
+                cy.createWithAPI('ITILValidationTemplate_Target', {
+                    itilvalidationtemplates_id: validationtemplates_id,
+                    itemtype: 'User',
+                    items_id: 2
+                });
+                cy.visit(`/front/ticket.form.php?id=${ticket_id}`);
+                cy.get('.timeline-buttons .main-actions button.dropdown-toggle-split').click();
+                cy.findByText('Ask for validation').click();
+                cy.get('.itilvalidation').within(() => {
+                    cy.getDropdownByLabelText('Template').selectDropdownValue(`test ${rand}`);
+                    cy.get('select[name="[validatortype]"]').should('have.value', 'User');
+                    cy.get('select[name="items_id_target"]').should('have.value', '2');
+                    cy.get('textarea[name="comment_submission"]').awaitTinyMCE().should('contain.text', 'test content');
+                });
+                cy.visit(`/front/ticket.form.php?id=${ticket_id}`);
+                cy.get('.timeline-buttons .main-actions button.dropdown-toggle-split').click();
+                cy.findByText('Ask for validation').click();
+                cy.get('.itilvalidation').within(() => {
+                    cy.getDropdownByLabelText('Template').selectDropdownValue(`test no approver ${rand}`);
+                    cy.get('select[name="[validatortype]"]').should('have.value', '0');
+                    cy.get('select[name="items_id_target"]').should('not.exist');
+                    cy.get('textarea[name="comment_submission"]').awaitTinyMCE().should('contain.text', 'no approver test content');
+                });
+            });
```

Comment:
Please use stronger selectors (accessibility based or with tests ids).

Reply:
Comment textarea cannot use label selectors because the input the label applies to is hidden and TinyMCE doesn't handle fixing the label.
Selects cannot use label selectors because there is a single label for both fields and this is an issue globally for itemtype/item fields as far as I am aware.

Reply:
> Comment textarea cannot use label selectors because the input the label applies to is hidden and TinyMCE doesn't handle fixing the label. 

I think `findByLabelText()` works for hidden items.

> Selects cannot use label selectors because there is a single label for both fields and this is an issue globally for itemtype/item fields as far as I am aware.

We can add the required aria labels in the dropdown code so it can be fixed.
I think it is better to fix these missing labels when needed to prevent adding brittle tests.

Reply:
> We can add the required aria labels in the dropdown code so it can be fixed. I think it is better to fix these missing labels when needed to prevent adding brittle tests.

We could, but there are more important things that needs fixed right now I think. At the very least, it should be done in a different PR and not block this PR. I do want to revisit accessibility issues more globally at some point.